File: api/Models/LearningResourceType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models;

public partial class LearningResourceType
{
    [Column("Id")]
    public long Id { get; set; }

    [Column("Name")]
    public string Name { get; set; } = null!;

    [Column("UnitType")]
    public string UnitType { get; set; } = null!;

    [Column("LearnerId")]
    public long LearnerId { get; set; }

    [Column("DeletedAt")]
    public DateTime? DeletedAt { get; set; }

    [Column("IsDeleted")]
    public bool IsDeleted { get; set; }

    public virtual Learner Learner { get; set; } = null!;

    public virtual ICollection<LearningResource> LearningResources { get; set; } = new List<LearningResource>();
}

File: mobile/app/src/main/java/com/example/purelearn/domain/model/LearningResourceType.kt
package com.example.purelearn.domain.model

data class LearningResourceType(
    val deletedAt: String,
    val id: Int,
    val learner: String,
    val learnerId: Int,
    val learningResources: List<String>,
    val name: String,
    val unitType: String
)